### PR TITLE
Disallow nonroot user from CUD column list profile

### PR DESCRIPTION
**Pull request by haneslinger**
non root view:
image
root view:
image

see also:
 - column list profile CUD
 - filter group CUD

### CODE REVIEW

**Comment by haneslinger** [comment]
@axelstudios these errors don't come the parent branch, they come from these commits. I've looked and the tests and the errors but I don't really understand them, can you help?

**Comment by haneslinger** [comment]
@kflemin re-pinging

File: seed/static/seed/partials/inventory_list.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h1>{$:: (inventory_type === 'taxlots' ? 'Tax Lots' : 'Properties') | translate
                     <option value="update_salesforce" ng-disabled="!organization.salesforce_enabled || selectedCount === 0" translate>Update Salesforce</option>
                 </optgroup>
                 <optgroup label="{$:: 'Options' | translate $}">
-                    <option value="open_show_populated_columns_modal" ng-disabled="data.length === 0" translate>Only Show Populated Columns</option>
+                    <option value="open_show_populated_columns_modal" ng-if="menu.user.is_ali_root" ng-disabled="data.length === 0" translate>Only Show Populated Columns</option>
```
Comment (kflemin) [inline seed/static/seed/partials/inventory_list.html]: is this getting hidden b/c it essentially updates a column list profile?

**Review by kflemin — APPROVED** [review]

File: seed/static/seed/partials/inventory_list.html (hunk 1 above)
Comment (haneslinger) [reply seed/static/seed/partials/inventory_list.html]: yep!

**Pull request merged**